### ISSUE TITLE
Hybrid Query doesn't respect Metric, NProbes and RefineFactor

### ISSUE THREAD

**Issue by edwin-jsi** [thebracket/LanceDbCSharp#20]
it will always set to 
        // Defaults
        DistanceMetric = LanceDbInterface.Metric.L2;
        NumProbes = 0;
        RefinementFactor = 0;

in internal VectorQueryBuilder(QueryBuilder parent) : base(parent.ConnectionId, parent.TableId)

**Comment by thebracket** [comment]
Good catch - I've set the query handler to respect them in the pass-through query.

**Comment by edwin-jsi** [comment]
the ToBatchesAsync has been fixed. however, the ToBatches has same issue.

**Comment by edwin-jsi** [comment]
the ToBatchesAsync in HybridQueryBuilder has been fixed. however, the ToBatches has same issue.

**Comment by thebracket** [comment]
Ok, I've added the same to ToBatches. Thanks!

**Comment by edwin-jsi** [comment]
verified.
